src/features/feeds/Feeds.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import {View} from 'react-native';
+import {Posts} from '../../components/list/Posts';
 
 import {feedsScreenStyles} from './feedsScreenStyles';
 
-export const Feeds = () => <View style={feedsScreenStyles.feedsScreen}></View>;
+export const Feeds = () => (
+  <View style={feedsScreenStyles.feedsScreen}>
+    <Posts />
```

Comment:
What is the reason split feed screen to posts component?
Lets keep Feed.tsx where we will have all screen layout and feed card component ...